riff-raff/app/views/preview/yaml/showTasks.scala.html

```diff
@@ -53,7 +53,7 @@ <h4 class="pull-left">
                 <div class="panel-body">
                     <ul class="magenta-list task-list">
                     @deploymentTasks.tasks.map { task =>
-                        <li><span class="preview-task"><strong>@task.name</strong> @task.description</span></li>
+                        <li><span class="preview-task"><strong>@task.name</strong> @task.verbose</span></li>
```

Comment:
is this a deliberate change?

Reply:
Yes - not sure the verbose is used in many places but it's often more descriptive. I'll review.

Reply:
So I killed `verbose` entirely. Well... not entirely since I didn't want to mess with persistance, but mostly.